.circleci/config.yml

```diff
@@ -113,75 +113,33 @@ jobs:
             xvfb-run -a pipenv run python3 test_gui.py
 
   staging-test-with-rebase:
-    docker:
-      - image: quay.io/freedomofpress/circleci-docker:latest
-        environment:
-          FPF_CI: true
-          CI_SD_ENV: staging
-          CI_AWS_TYPE: t2.medium
-          FPF_GRSEC: false
-          TEST_REPORTS: /root/sd
-    working_directory: ~/sd
+    machine:
+      enabled: true
 
+    working_directory: ~/sd
     steps:
-
       - checkout
 
       - run:
           name: Rebase on-top of github target
           command: ./devops/scripts/rebase-ci.sh
 
       - run:
-          name: Installation apt pre-reqs
-          command: apt-get install -y enchant
-
-      - run:
-          name: Install pip==18.0 to work pipenv/pip bug
-          command: pip install pip==18.0
-
-      - run:
-          name: Installation pre-reqs
-          command: pip install -U -r securedrop/requirements/develop-requirements.txt
-
-      - run:
-          name: Check Python dependencies for CVEs
-          command: make safety
-
-      - run:
-          name: Run static security testing on source code
-          command: make bandit
```

Comment:
was removing these steps (static analysis and checking for CVEs) intentional? 

ok if not (I will add them again), just was attempting to make some unrelated CI changes over in `xenial-pgp-journalist` and noticed this

Reply:
No, that was _not_ intentional, @redshiftzero — my oversight during review. I'll open an issue to track the re-add, fine if it comes in via your Xenial-related CI changes.